document/core/valid/types.rst

```diff
@@ -146,28 +146,26 @@ Memory Types
    }
 
 
-.. index:: event type, exception attribute, result type
-   pair: validation; event type
-   single: abstract syntax; event type
-.. _valid-eventtype:
+.. index:: exception type, function type
+   pair: validation; exception type
+   single: abstract syntax; exception type
+.. _valid-exntype:
 
-Event Types
-~~~~~~~~~~~
-
-:math:`\EXCEPTION~[t^n] \to []`
-...............................
+Exception Types
+~~~~~~~~~~~~~~~
 
-* The :ref:`event attribute <syntax-eventattr>` must be |EXCEPTION|.
+:math:`[t^n] \to []`
+....................
 
 * The :ref:`function type <syntax-functype>` :math:`[t^n] \to []` must be :ref:`valid <valid-functype>`.
 
```

Comment:
```suggestion
* The :ref:`function type <syntax-functype>` :math:`[t_1^n] \to [t_2^m]` must be :ref:`valid <valid-functype>`.

* The type sequence :math:`t_2^m` must be empty.
```

Reply:
Again, not sure why I missed putting this in the batch of suggestions to apply. :grimacing: Applying it in the next commit.